tests/phpunit/unit-tests/test-notices.php

```diff
@@ -159,4 +159,13 @@ public function test_html_notice() {
 
 		$this->assertNotFalse( strpos( $html, $form ) );
 	}
+
+	/* Test empty $this->errors and $this->notices */
+	public function test_gform_admin_messages_hooks() {
```

Comment:
I think we need to enhance the tests here to check specifically that the registered errors/notices are returned on Gravity PDF pages that run the `gform_admin_error_messages` or `gform_admin_messages` filters, but are NOT returned if the pages are not Gravity PDF ones.

Reply:
You can use `apply_filters()` to trigger the correct filters, but you'll need to override the globals to mock Gravity PDF vs non-Gravity PDF pages.